source/Mlos.Python/mlos/Optimizers/SimpleBayesianOptimizer.py

```diff
@@ -300,18 +300,6 @@ def estimate_local_parameter_importance(self, params, t=None):
 
         return local_parameter_importance
 
-    def optimum(self, stay_focused=False):

```

Comment:
I removed some functionality here but this class will be removed anyway from what I understand.

Reply:
Yeah, I wouldn't worry about it. The more we break it before deleting it, the less likely it is that someone will take a dependency on it :D